server/http/restapi/asset.go

```diff
@@ -0,0 +1,56 @@
+package restapi
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+func (a *restapi) handleGetStaticAsset(w http.ResponseWriter, req *http.Request, actingUserID string) {
+	vars := mux.Vars(req)
+
+	assetName := vars["name"]
+	appID := vars["app_id"]
+
+	if appID == "" {
+		httputils.WriteBadRequestError(w, errors.New("app_id not specified"))
+		return
+	}
+
+	if assetName == "" {
+		httputils.WriteBadRequestError(w, errors.New("asset name not specified"))
+		return
+	}
+
+	// TODO verify that request is from the correct app
+
+	resp, err := a.api.Proxy.GetAsset(apps.AppID(appID), assetName)
+	if err != nil {
+		httputils.WriteBadRequestError(w, err)
+		return
+	}
+
+	copyHeaders(w.Header(), resp.Header)
```

Comment:
Do we really need that? For AWS apps `resp.Header` is `nil` anyway. By dropping that line AWS and HTTP act the same which is what we want here, right?

Reply:
Actually I'm 0/5. I still left headers there, just in case. AWS will not have any headers but HTTP apps might. I don't know if we would ever need them though.

Reply:
I'm 2/5 that we can add the header later if we need them. (Maybe as a whitelist?) For now, we should we strict as dropping them at some point is harder.

Reply:
Agree.